packages/client/src/runtime/utils/date.ts

```diff
@@ -0,0 +1,7 @@
+export function isDate(value: unknown): value is Date {
+  return Object.prototype.toString.call(value) === '[object Date]'
```

Comment:
There's a few other places in the client that either have this check hardcoded or contain a similar `isDate` function, it would be nice to de-duplicate them and make them use this one.

Reply:
it's fine to do that in another PR though